smoreo:
  camera_height_from_ground: 0.861
  cone_height: 0.38
  cut_off_y: 1000
  cx: 740.0
  cy: 560.0
  f: 750.0
